presto-main/src/main/java/com/facebook/presto/testing/MaterializedResult.java

```diff
@@ -69,7 +69,7 @@
 public class MaterializedResult
         implements Iterable<MaterializedRow>
 {
-    public static final int DEFAULT_PRECISION = 5;
+    public static final int DEFAULT_PRECISION = 4;
```

Comment:
Some of our tests failed because of double aggregation, and this seemed like the most reasonable solution. However, I later changed the tests to not do that, so I'll revert this.